internal/saucecloud/cloud.go

```diff
@@ -336,6 +340,13 @@ func (r *CloudRunner) runJobs(jobOpts chan job.StartOptions, results chan<- resu
 			continue
 		}
 
+		if opts.Attempt == opts.MaxAttempt && opts.PassCount < opts.MinPass {
```

Comment:
Sorry, I introduce that error while doing the suggestion, it should be `!=` not `==`.
Otherwise, behavior is not the one expected.

```suggestion
		if opts.Attempt != opts.MaxAttempt && opts.PassCount < opts.MinPass {
```